maat-court-data-api/src/main/java/gov/uk/courtdata/dto/PassportAssessmentEvidenceDTO.java

```diff
@@ -0,0 +1,29 @@
+package gov.uk.courtdata.dto;
+
+import lombok.AllArgsConstructor;
+import lombok.Builder;
+import lombok.Data;
+import lombok.NoArgsConstructor;
+
+import java.io.Serializable;
+import java.time.LocalDateTime;
+
+@Data
+@Builder
+@AllArgsConstructor
+@NoArgsConstructor
+public class PassportAssessmentEvidenceDTO implements Serializable {
+    private Integer id;
+    private LocalDateTime dateReceived;
+    private LocalDateTime dateCreated;
+    private String userCreated;
+    private LocalDateTime dateModified;
+    private String userModified;
+    private String active;
```

Comment:
The fields here such as `active`, `mandatory` and `adhoc` that are kept as Y/N Strings on the database, were we taking the approach of having them as Booleans in the DTOs and using a convertor to translate the values?

Reply:
Ok that makes sense, happy to resolve as we need to keep them both in line to make the logic in orchestrator simpler.